server/middleware/setUpWebSession.ts

```diff
@@ -21,7 +21,7 @@ export default function setUpWebSession(): Router {
     session({
       store,
       name: 'hmpps-sentence-plan-ui.session',
-      cookie: { secure: config.https, sameSite: 'lax', maxAge: config.session.expiryMinutes * 60 * 1000 },
+      cookie: { secure: config.https, sameSite: 'Strict', maxAge: config.session.expiryMinutes * 60 * 1000 },
```

Comment:
this needs to be all-lowercase `strict`

Reply:
(see https://github.com/ministryofjustice/hmpps-sentence-plan-ui/actions/runs/13836930866/job/38714239108?pr=436#step:7:129)